lib/open_project/static/links.rb

```diff
@@ -268,6 +268,9 @@ def static_links
               },
               status_read_only: {
                 href: "https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-status/#create-a-new-work-package-status"
+              },
+              customize_life_cycle: {
+                href: "https://www.openproject.org/enterprise-edition" # TODO: update
```

Comment:
Just a reminder that the target page is still pending and should be inserted here later.

Reply:
Already created https://community.openproject.org/wp/60343 to not forget

cc @MayaBerd